### ISSUE TITLE
VHosts and base directory

### ISSUE THREAD

**Issue by zzeesstt** [teddysun/lamp#89]
When creating a new vhosts, the entire redirect from the ip address goes to /data/www/new_vhost's.
How to leave access through ip to the base vhost's /data/www/default?
I can not find in which configuration file I can configure it.

**Comment by teddysun** [comment]
Default Virtual Host conf
/usr/local/apache/conf/extra/httpd-vhosts.conf

**Comment by zzeesstt** [comment]
@teddysun, Yes, but I do not have to edit it so that all users logging in to the site on ip fall into a certain vhosts
